engine/menus/pokedex.asm

```diff
@@ -81,7 +81,7 @@ HandlePokedexSideMenu:
 	ld hl, wTopMenuItemY
 	ld a, 10
 	ld [hli], a ; top menu item Y
-	ld a, 15
+	ld a, 15 ; in other localizations, this is 12
```

Comment:
This will be relevant when/if other localizations are integrated into pret/pokered (issue #271).